database/ringgold/V4__ringgold_data.2016-01.sql

```diff
@@ -127,6 +127,14 @@ INSERT INTO ringgold.institutions (rec_id,parent_ringgold_id,ringgold_id,name,ci
 INSERT INTO ringgold.institutions (rec_id,parent_ringgold_id,ringgold_id,name,city,post_code,country,state,type,timestamp) 
     VALUES (1000006,0,1000006,'Test Group1 I006','Paris','75480','FR','','academic/health','2014-08-04 10:44:31');
 
+/* institution names which contain a single quote(s) */
```

Comment:
I thought about that. migrations should be treated as immutable things. ok, let me create a separate migration. the ringgold migrations are only used for testing (in docker).